### ISSUE TITLE
Exotel Support

### ISSUE THREAD

**Issue by abhishekelectricspeech** [voxos-ai/bolna#383]
Hi ,

I can see exotel as a provider as code, but I don't see that option available.
So you plan to add it and if yes, Is there a timeline?

**Comment by marmikcfc** [comment]
Do you mean on the dashboard?

**Comment by abhishekelectricspeech** [comment]
Yes , is it available on dashboard , and If not can i integrate with it without dashboard , do you have any guides on it ?
